lab-Jp/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "lab-01-modular-patterns-and-testing",
+  "version": "1.0.0",
+  "description": "[![Build Status](https://travis-ci.org/codefellows-sea-js-401d8-assignments/lab-01-modular-patterns-and-testing.svg?branch=master)](https://travis-ci.org/codefellows-sea-js-401d8-assignments/lab-01-modular-patterns-and-testing)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Firefly4266/lab-01-modular-patterns-and-testing.git"
+  },
+  "keywords": [],
+  "author": "\"jp Gaines\"",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Firefly4266/lab-01-modular-patterns-and-testing/issues"
+  },
+  "homepage": "https://github.com/Firefly4266/lab-01-modular-patterns-and-testing#readme",
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "gulp": "^3.9.1",
+    "gulp-eslint": "^3.0.1",
+    "gulp-mocha": "^2.2.0",
+    "mocha": "^2.5.3"
+  },
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "chai": "^3.5.0"
```

Comment:
We are using chai as a dev dependency but for some reason npm init adds it to dependencies.  I just left it as I did't think it was hurting anything for one, secondly I was assuming npm needed it there since it put it there.